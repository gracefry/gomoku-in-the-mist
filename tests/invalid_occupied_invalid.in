who
place A30
who
place A1
who
place A1
who
place P10
who
place B1
who
place A2
place B2
place A3
place B3
place A4
place B4
place A5