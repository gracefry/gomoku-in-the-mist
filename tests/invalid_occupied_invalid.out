B
Invalid coordinate
B
W
Occupied coordinate
W
Invalid coordinate
W
B
Black wins!
A1B1A2B2A3B3A4B4A5
Thank you for playing!